pkg/kubelet/container/runtime.go

```diff
@@ -97,6 +97,8 @@ type Runtime interface {
 	RemoveImage(image ImageSpec) error
 	// Returns Image statistics.
 	ImageStats() (*ImageStats, error)
+	// Cadvisor FS Label
+	ImageLabel() string
```

Comment:
@sjpotter Actually, does this has to be implemented by runtime? Can it be implemented by kubelet?
e.g. when kubelet starts, it should be able to tell what runtime to use, so then it can save the image label in a variable.